tools/l2t_scaffolder.py

```diff
@@ -0,0 +1,24 @@
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+# pylint: disable=unused-argument
+# pylint: disable=no-value-for-parameter
+# Docstrings are used by click in generating output. Since we don't want this
+#  behavior the following functions do not contain docstrings.
+
+"""The l2t scaffolder tool."""
+import click
+
+from plasoscaffolder.frontend import cli
+
+
+@click.command()
+@click.option(
+    '--definition', envvar='SCAFFOLDER_DEFINITION', default='',
+    help='Project or definition to choose.', callback=cli.ScaffolderCli.Start)
+def StartCli(definition):
+  pass
```

Comment:
This is very strange - why is the command code in an option callback, and not the startcli method here? This makes all the functionality a bit weird, like having _OUTPUT_HANDLER be a class constant in the frontend, instead of an instance variable like I'd expect.
Is there some reason for using a callback here that I'm missing?